packages/rocknix/config/system/configs/system.cfg

```diff
@@ -178,6 +178,7 @@ syncthing.enabled=0
 system.autohotkeys=1
 system.automount=1
 system.battery.warning=0
+system.battery.warning=25
```

Comment:
You have duplicated this setting?

Reply:
my bad, it should be `system.battery.warning_threshold=25`

I'll fix this